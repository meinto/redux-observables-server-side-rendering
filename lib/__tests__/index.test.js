import * as indexExports from '../index'

describe('index file tests', () => {
  it('snapshots exports', () => {
    expect(indexExports).toMatchSnapshot()
  })
})