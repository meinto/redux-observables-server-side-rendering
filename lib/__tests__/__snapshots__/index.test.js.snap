// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index file tests snapshots exports 1`] = `
Object {
  "SSR": [Function],
  "SSR_DEPENDENCIES_MOCK": Object {
    "notFound": [Function],
    "observe": [Function],
    "redirect": [Function],
  },
}
`;
